site/source/tutorials/008_animations/index.md

```diff
@@ -0,0 +1,370 @@
+
+{% aside 'Meta' %}
+Dojo 2 meta provides a means to get and set properties against the generated HTML without exposing the domNode itself.
+{% endaside %}
+
+{% task 'Animating our zombies' %}
+
+To animate the zombies, we use the `@dojo/widget-core/meta/WebAnimation` meta. This takes a `key` and `AnimationProperties`.
+Lets add animation properties to our first Zombie.
+
+{% instruction 'Add the following to the render function in Zombies.ts' %}
+
+``` typescript
+// add the import
+import WebAnimation from '@dojo/widget-core/meta/WebAnimation';
+
+//add to the top of the render function
+const zombieOneMoveAnimation = {
+	id: 'zombieOneMove,
+	effects: [
+		{ left: '0%' },
+		{ left: '35%' }
+	],
+	timing: {
+		duration: 2000,
+		easing: 'ease-in',
+		fill: 'forwards'
+	},
+	controls: {
+		play: true
+	}
+};
+
+this.meta(WebAnimation).animate('zombieOne', zombieOneMoveAnimation);
+```
+
+{% aside 'Reminder' %}
+If you cannot see the zombies, make sure you have run `dojo build -w` to build the application and start the development server.
+{% endaside %}
+
+Refresh your browser and you should now see the left zombie moving across the screen for 2 seconds.
+
+So let's talk about the animation properties we've created here:
+
+- A unique id for this animation
+- Effects array depicts the steps for the animation, in this case we're animating the `left` style from `0%` to `35%`
+- The timing object specifies the duration and easing effects for the animation. The fill param specifies that the animation should finish in it's final position
+- The controls object in this case simply sets play to `true`
+
+{% task 'Animating our zombie on click' %}
+
+To make the zombie animate on click, we need to use our widgets state to control the `play` property of the `zombieOneMoveAnimation` object. We can do this by creating a `_play: boolean` property on our widget and toggling it in our zombie click function.
+
+{% instruction 'Edit Zombies.ts' %}
+
+``` typescript
+export class Zombies extends WidgetBase {
+	// add the _play boolean at the top of the class
+	private _play = false;
+
+	private _onZombieClick() {
+		// add the toggle and an invalidate call to the zombie
+		// click function
+		this._play = !this._play;
+		this.invalidate();
+	}
+
+	// now use this._play instead of a hardcoded `true` in the `zombieOneMoveAnimation` object
+```
+
+Refresh the browser and the zombie animation should now play / pause when clicked.
+
+{% section %}
+
+## Animation events
+
+The webanimation meta provides callback functions for `onFinish` and `onCancel`. To ensure that we can replay our animation we need to set `this._play` to false when the animation has finished, to do this we will add an `onFinish` callback.
```

Comment:
has finished, to do this we will add an `onFinish` callback. -> has finished by adding an `onFinish` callback.